### PR TITLE
features: aaa: add FAQ

### PR DESCRIPTION
**Pull request by machacekondra**
This patch creates page which summarize some of the most common asked questions for AAA.

### CODE REVIEW

File: source/develop/release-management/features/infra/aaa_faq.html.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+### Is it possible to use client certificate authentication?
+
+Yes, it's possible with *apache mod_ssl* module. Full blog post is available [here](http://machacekondra.blogspot.cz/2016/02/client-certificate-authentication-with.html).
+
+### Is it possible to use CAS for authentication?
+
+Yes, it's possible with *apache mod_auth_cas* module. Full blog post is available [here](http://www.ovirt.org/blog/2016/04/sso/).
+
+### Is it possible to use SSD for authentication?
```
Comment (mwperina) [inline source/develop/release-management/features/infra/aaa_faq.html.md]: SSD -> SSSD

Comment (machacekondra) [inline source/develop/release-management/features/infra/aaa_faq.html.md]: Done.

File: source/develop/release-management/features/infra/aaa_faq.html.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+### Is it possible to use CAS for authentication?
+
+Yes, it's possible with *apache mod_auth_cas* module. Full blog post is available [here](http://www.ovirt.org/blog/2016/04/sso/).
+
+### Is it possible to use SSSD for authentication?
+
+Unfortunatelly it's not supported, yet. You can track [this](https://bugzilla.redhat.com/show_bug.cgi?id=829292) bugzilla for more information.
+
+<Category:Feature>
```
Comment (mwperina) [inline source/develop/release-management/features/infra/aaa_faq.html.md]: Looks good to me. @sandrobonazzola, @thatdocslady, @mykaul  could you please merge?

**Comment by machacekondra** [comment]
Added jks password change question.

**Comment by mwperina** [comment]
Looks good to me. @sandrobonazzola, @thatdocslady, @mykaul could you please merge?

**Comment by mykaul** [comment]
Waiting for the Travis CI issues to be resolved - otherwise, :+1:  from me - will merge when it passes CI (probably the failure is unrelated, but rather not add more issues to the pipeline)

**Comment by machacekondra** [comment]
Travis errors are unrelated to this patch.

**Pull request merged**